src/main/kotlin/team/kin/forest/domain/user/adapter/input/AuthWebAdapter.kt

```diff
@@ -43,4 +38,14 @@ class AuthWebAdapter(
             .let { authDataMapper toResponse it }
             .let { ResponseEntity.ok(it) }
 
+    @PostMapping("/send/phone-number/{phoneNumber}")
+    fun sendAuthCode(@PathVariable phoneNumber: String): ResponseEntity<Void> =
```

Comment:
```suggestion
    @PostMapping("/send/phone-number/{phone_number}")
    fun sendAuthCode(@PathVariable("phone_number") phoneNumber: String): ResponseEntity<Void> =
```
이렇게 작성하는게 좋을 거 같아요~!